CONTRIBUTING.md

```diff
@@ -72,6 +72,99 @@ and indicate which issue is fixed.
+
+### Prepare your environment
+
+Clone and setup [RED4.RTTIDumper](https://github.com/WopsS/RED4.RTTIDumper), read through the [build](https://github.com/WopsS/RED4.RTTIDumper?tab=readme-ov-file#build-instructions) 
+instructions.
+Make sure to update RED4ext.SDK dependency, in `<RTTIDumper>/deps/red4ext.sdk` to use the latest snapshot of the SDK.
+You can comment each line adding a [custom writer](https://github.com/WopsS/RED4.RTTIDumper/blob/4feaad12de106e395b7a637031432ea3caa2cf10/src/Main.cpp#L29).
+This way, it will only generate the C++ dump and make the process faster.
+
+### Declare a new member
+
+All operations below are done within `<RTTIDumper>/deps/red4ext.sdk`, in order for the dumper to account for the change
+you want to write. We will see the content of your PR after this step.
+
+We will declare a new member of the class [vehicleBaseObject](https://github.com/WopsS/RED4ext.SDK/blob/fa50370bc0c4d95c16b8ec9e0a10113f155f361f/include/RED4ext/Scripting/Natives/Generated/vehicle/BaseObject.hpp).
+In this example, it will be the property `bool isOnGround` which was found out at offset `0x25C`.
+If you look at the class, the file is in `../include/RED4ext/Scripting/Natives/Generated/vehicle/BaseObject.hpp`.
+
+We need to declare this file to tell the SDK we want to use our modified version of this class, instead of the generated
+one:
+1. Copy this file in `../include/RED4ext/Scripting/Natives/`.
+2. Rename the file using the full name of the class, which will be [vehicleBaseObject.hpp](https://github.com/WopsS/RED4ext.SDK/blob/fa50370bc0c4d95c16b8ec9e0a10113f155f361f/include/RED4ext/Scripting/Natives/Generated/vehicle/BaseObject.hpp#L20):
+
+```cpp
+    static constexpr const char* NAME = "vehicleBaseObject";
+```
+
+3. Remove `__declspec()` declaration:
+
+```diff
+- struct __declspec(align(0x10)) BaseObject : game::Object
++ struct BaseObject : game::Object
+```
+
+4. Declare the new member based on its offset and type:
+
+```diff
+{
+    static constexpr const char* NAME = "vehicleBaseObject";
+    static constexpr const char* ALIAS = "VehicleObject";
+
+-   uint8_t unk240[0x3A0 - 0x240]; // 240
++   uint8_t unk240[0x25C - 0x240]; // 240
++   bool isOnGround;               // 25C
++   uint8_t unk25D[0x3A0 - 0x25D]; // 25D
+    Ref<AI::Archetype> archetype;  // 3A0
+    uint8_t unk3B8[0x6D2 - 0x3B8]; // 3B8
+    bool isVehicleOnStateLocked;   // 6D2
+    uint8_t unk6D3[0xB90 - 0x6D3]; // 6D3
+};
+```
+
+5. Assert the offset of known members:
+
+```diff
+};
+RED4EXT_ASSERT_SIZE(BaseObject, 0xB90);
++RED4EXT_ASSERT_OFFSET(BaseObject, isOnGround, 0x25C);
++RED4EXT_ASSERT_OFFSET(BaseObject, archetype, 0x3A0);
++RED4EXT_ASSERT_OFFSET(BaseObject, isVehicleOnStateLocked, 0x6D2);
+} // namespace vehicle
+using vehicleBaseObject = vehicle::BaseObject;
+using VehicleObject = vehicle::BaseObject;
+} // namespace RED4ext
+```
+
+### Generate stub
+
+1. Build the library and install it in `<game>/red4ext/plugins/`.
```

Comment:
I would say something like `Build the RTTIDumper` with the modified SDK and install ...`

Reply:
Should be good.